components-core/src/main/java/com/adyen/checkout/components/core/internal/ui/model/SessionParams.kt

```diff
@@ -0,0 +1,19 @@
+/*
+ * Copyright (c) 2023 Adyen N.V.
+ *
+ * This file is open source and available under the MIT license. See the LICENSE file for more info.
+ *
+ * Created by josephj on 7/3/2023.
+ */
+
+package com.adyen.checkout.components.core.internal.ui.model
+
+import androidx.annotation.RestrictTo
+import com.adyen.checkout.components.core.Amount
+
+@RestrictTo(RestrictTo.Scope.LIBRARY_GROUP)
+interface SessionParams {
```

Comment:
Do we really need this to be an interface? Can't we use the same data class for testing and other things?

Reply:
Hmm yeah looking at it now it doesn't make much sense anymore